astropy/wcs/wcsapi/utils.py

```diff
@@ -74,11 +74,9 @@ def wcs_info_str(wcs):
 
     # Find largest between header size and value length
     world_dim_width = max(9, len(str(wcs.world_n_dim)))
-    world_nam_width = max(
-        9, max(len(x) if x is not None else 0 for x in wcs.world_axis_names)
-    )
+    world_nam_width = max(9, *(len(x) for x in wcs.world_axis_names if x is not None))
     world_typ_width = max(
-        13, max(len(x) if x is not None else 0 for x in wcs.world_axis_physical_types)
+        0, 13, *(len(x) for x in wcs.world_axis_physical_types if x is not None)
```

Comment:
Why is the `0,` added here?

Reply:
Just guaranteeing a minimum of 2 arguments.
Most of the time wcs.world_axis_physical_types is an iterable of None, so it would become ``max(13)``, which throws an error. By putting in `0` it then becomes ``max(0, 13, ...)`` which is fine.

Reply:
Hmm, that does not feel great. How about just `max(13, *(len(x or "") for x in wcs.world_axis_physical_type))`. (or back to `len(x) if x is not None else 0`). Admittedly, that is slower (second one a bit better), but if it is all about speed then my tests suggests one should do
```
max([13] + [len(x) for x in wcs.world_axis_physical_types if x is not None])
```

Reply:
Another option would be 
```suggestion
    try:
        world_typ_width = max(
            13, *(len(x) for x in wcs.world_axis_physical_types if x is not None)
        )
    except TypeError:
        world_typ_width = 13  # world_axis_physical_types was all None.
```
But I haven't timed this so I can't tell if this is faster or slower than the other suggestions.

Reply:
Did ``max([13] + [len(x) for x in wcs.world_axis_physical_types if x is not None])`` as the fastest of the 1-liners.